foundations/foundation-config/src/main/java/org/apache/servicecomb/config/priority/ConfigObjectFactory.java

```diff
@@ -205,7 +211,7 @@ private PriorityProperty<?> createIntProperty(Field field, String[] keys, Intege
 
   private String[] collectPropertyKeys(Field field, String prefix, Map<String, Object> parameters) {
     String propertyPrefix = prefix;
-    String[] keys = new String[] {field.getName()};
+    String[] keys = new String[]{field.getName()};
```

Comment:
checkstyle changes? Why not detected by checktsyle?

Reply:
should I add a space?

Reply:
use formatter in etc foler and then format code